packages/qdrant-loader-core/src/qdrant_loader_core/graph/falkor_store.py

```
@@ -0,0 +1,364 @@
+
+        edges = [{"source": row[0], "target": row[1]} for row in edge_result.result_set]
+
+        return nodes, edges
+
+    # -------------------------
+    # Batch update cluster_id
+    # -------------------------
+    async def update_clusters_batch(
+        self, updates: list[dict], project: str | None = None
+    ):
+        async def _run(row):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_run`

Add return type annotation: `None`

(ANN202)